demos/signal-processing/Dockerfile-nifi

```diff
@@ -1,3 +1,5 @@
-FROM docker.stackable.tech/stackable/nifi:1.27.0-stackable24.7.0
+# docker build -f ./Dockerfile-nifi -t docker.stackable.tech/demos/nifi:1.27.0-postgresql .
+
+FROM docker.stackable.tech/stackable/nifi:1.27.0-stackable24.11.0
```

Comment:
```suggestion
FROM docker.stackable.tech/stackable/nifi:1.27.0-stackable24.7.0
```

Reply:
Or maybe Nifi 2.0.0 (as suggested by @sbernauer)?

Reply:
Actually, we won't do this now because we use the demo update process as part of release process testing - so we should use the latest stable version.

It can be bumped after critical release tasks are done though.